src/main/java/seedu/exchangecoursemapper/ui/UI.java

```diff
@@ -14,6 +14,11 @@ public void displayGreeting() {
         System.out.println(greetingMessage);
     }
 
+    public void displayExitMessage() {
+        String exitMessage = "All the best in planning for your exchange, hope we helped!";
+        System.out.println(exitMessage);
+    }
+
```

Comment:
I like that you extracted out the user interaction for the goodbye message into the UI class for better cohesion!